components/org.wso2.carbon.identity.password.policy/src/main/java/org/wso2/carbon/identity/password/policy/handler/PasswordPolicyValidationHandler.java

```diff
@@ -51,6 +52,11 @@
     @Override
     public void handleEvent(Event event) throws IdentityEventException {
 
+        // Skip password policy validation if the Ask Password is enabled for the flow.
```

Comment:
Update the comment : 
This thread-local denotes to skip the password pattern validation. Ex. Ask password flow, JIT flow

Reply:
Fixed